### PR TITLE
[IMP] mail: composer resize

### PR DESCRIPTION
**Pull request by tsm-odoo**

### CODE REVIEW

**Comment by alexkuhn** [comment]
When height of page is small and content in composer is long, cannot scroll to very bottom of composer 
unable-to-scroll-chatter-composer

Also the scroll does not work when just below the `max-height`.

**Comment by alexkuhn** [comment]
Also, out-of-scope of PR and also present in master:
chat-window-composer-long-text-not-pretty-transition-top

Something that we can improve at a later date. Just posting for awareness of issue.

File: addons/mail/static/src/composer/composer.js [hunk 1]
```diff
@@ -27,6 +27,14 @@ export class Composer extends Component {
...
             this.state.value = message ? this.convertBrToLineBreak(message.body) : "";
         });
+        useEffect(
+            () => {
+                this.ref.el.style.height = "1px";
+                this.ref.el.style.height = this.ref.el.scrollHeight + "px";
+            },
+            () => [this.state.value, this.ref.el]
+        );
+        onMounted(() => this.ref.el.scrollTo({ top: 0, behavior: "instant" }));
```
Comment (alexkuhn) [inline addons/mail/static/src/composer/composer.js:37]: What's the purpose of this `onMounted`?

Comment (tsm-odoo) [reply addons/mail/static/src/composer/composer.js:37]: Mimicking master behavior, i.e. having the scroll bar to the top when message edit starts.

Comment (alexkuhn) [reply addons/mail/static/src/composer/composer.js:37]: Could you make a test for this? This LOC was not clear by itself, so coverage with a test would make it clearer we need this LOC.

Comment (tsm-odoo) [reply addons/mail/static/src/composer/composer.js:37]: Here you go, should be fine now :+1:

**Review by alexkuhn — APPROVED** [review]

**Pull request merged**